static/js/components/tiles/line_tile.tsx

```diff
@@ -152,8 +169,7 @@ export function draw(
     chartData.unit
   );
   if (!isCompleteLine) {
-    svgContainer.querySelectorAll(".dotted-warning")[0].className +=
-      " d-inline";
+    elem.querySelectorAll(".dotted-warning")[0].className += " d-inline";
```

Comment:
maybe should check that querySelectorAll returns a non-empty list before accessing the first element? & maybe should also iterate through the list and add the class name to all selected elements?